README.md

```diff
@@ -9,7 +9,7 @@
 kind is a tool for running local Kubernetes clusters using Docker container "nodes".  
 kind is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
 
-If you have [go] and [docker] installed `go get -u sigs.k8s.io/kind && kind create cluster` is all you need!
+If you have [go] and [docker] installed `GO111MODULE="on" go get -u sigs.k8s.io/kind@master && kind create cluster` is all you need!
```

Comment:
we should drop @master in most places once we have a new tag up. v0.2.1 has the golint path issue